mocks/mock_application.py

```diff
@@ -54,9 +54,12 @@ def __init__(self, env='dev', **kwargs):
 
         if self['app_type'] == 'packaged':
             self['app_path'] = self._get_resource_path('packaged_app.zip')
+        elif self['app_type'] == 'xss_app':
+            self['url'] = 'http://%s.testmanifest.com/manifest.webapp' % "xssapp"
```

Comment:
Why do we need a different manifest url for this app? Can we not just use the one that uses a timestamp?